packages/preact/src/index.ts

```diff
@@ -358,12 +358,43 @@ export function useComputed<T>(compute: () => T) {
 	return useMemo(() => computed<T>(() => $compute.current()), []);
 }
 
+let oldNotify: (this: Effect) => void,
+	queue: Array<Effect> = [],
+	isFlushing = false;
+
+function flush() {
+	batch(() => {
+		let flushing = [...queue];
+		isFlushing = false;
+		queue.length = 0;
+		for (let i = 0; i < flushing.length; i++) {
+			oldNotify.call(flushing[i]);
+		}
+	});
+}
+
+function notify(this: Effect) {
+	queue.push(this);
+	if (!isFlushing) {
+		isFlushing = true;
+		(process.env.TEST
```

Comment:
Goal is to remove this and use an afterFrame helper in the tests when this is in the finalization phase